docs/environment-variables.md

```diff
@@ -8,52 +8,53 @@ This document outlines environment variables that can be used to customize behav
+| `ARGO_PROGRESS_FILE_TICK_DURATION`       | `time.Duration`      | `3s`                         | How often the progress file is read by the executor. Set to 0 to disable self reporting progress.                                                                                                                                                                        |
+| `ARGO_REMOVE_PVC_PROTECTION_FINALIZER`   | `bool`               | `true`                       | Remove the `kubernetes.io/pvc-protection` finalizer from persistent volume claims (PVC) after marking PVCs created for the workflow for deletion, so deleted is not blocked until the pods are deleted.  [#6629](https://github.com/argoproj/argo-workflows/issues/6629) |
+| `ARGO_TRACE`                             | `string`             | ``                           | Whether to enable tracing statements in Argo components.                                                                                                                                                                                                                 |
+| `ARGO_AGENT_PATCH_RATE`                  | `time.Duration`      | `DEFAULT_REQUEUE_TIME`       | Rate that the Argo Agent will patch the workflow task-set.                                                                                                                                                                                                               |
+| `ARGO_AGENT_CPU_LIMIT`                   | `resource.Quantity`  | `100m`                       | CPU resource limit for the agent.                                                                                                                                                                                                                                        |
+| `ARGO_AGENT_MEMORY_LIMIT`                | `resource.Quantity`  | `256m`                       | Memory resource limit for the agent.                                                                                                                                                                                                                                     |
+| `ARGO_POD_STATUS_CAPTURE_FINALIZER`      | `bool`               | `false`                      | The finalizer blocks the deletion of pods until the controller captures their status.                                                                                                                                                                                    | 
+| `ARGO_POD_STATUS_CAPTURE_TIMEOUT`        | `time.Duration`      | `10m`                        | After the timeout time, workflow controller will remove the finalizer anyway to avoid a potential risk of pods increased infinitely.                                                                                                                                     |
```

Comment:
this is the one line that was added here, for reference. can set "Hide whitespace" in GH diff to properly see that.